dbms/src/Flash/ResourceControl/LocalAdmissionController.cpp

```diff
@@ -0,0 +1,27 @@
+// Copyright 2023 PingCAP, Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <Flash/ResourceControl/LocalAdmissionController.h>
+
+namespace DB
+{
+
+#ifndef DBMS_PUBLIC_GTEST
+std::unique_ptr<LocalAdmissionController> LocalAdmissionController::global_instance;
+#else
+auto LocalAdmissionController::global_instance = std::make_unique<MockLocalAdmissionController>();
+#endif
+
+const std::string ResourceGroup::DEFAULT_RESOURCE_GROUP_NAME = "default";
```

Comment:
If resource-group-name in dag-request is empty, will it be assigned "default" in tiflash?

Reply:
tiflash should can handle resource_group_name is empy.

Reply:
if resource_group_name is empty, it means resource control is disabled, we should not ask GAC.
LAC should just ignore it(`getPriority() just return 1; computeResource() should do nothing and isResourceGroupThrottled() should return false`)

Reply:
todo next LAC pr will handle this!